src/api-binder.coffee

```diff
@@ -245,6 +250,28 @@ module.exports = class APIBinder
 				body: updatedFields
 			.timeout(conf.apiTimeout)
 
+	_sendLogsRequest: (uuid, data) =>
+		reqBody = _.map data, (msg) ->
+			return _.mapKeys(msg, (v, k) -> _.snakeCase(k))
```

Comment:
NIT: No need for `return` in CS no? Even if migrated to JS, you can use the `() => ( ... )` short-hand to avoid it

Reply:
Done